website/docs/routes/university.mdx

```diff
@@ -185,6 +185,40 @@
   | -------- | -------- | -------- | -------- | -------- | -------- |
   | xw       | zytzyyg  | xzcg     | xzjz     | xz       | bshzs    |
 
+## 北京师范大学-香港浸会大学联合国际学院 UIC {#bnu-hkbu-uic}
+
+:::tip
+这b学校的网站证书有问题（证书链不完整）导致 Linux 下的部分软件 curl / nodejs fetch / python requests 等无法验证其证书。
+
+[myssl.com](https://myssl.com/uic.edu.cn?domain=uic.edu.cn&status=q)
+
+需要设置 [NODE_TLS_REJECT_UNAUTHORIZED = '0'](https://github.com/DIYgod/RSSHub/commit/741a3b8a8e0e2fa8f347a0443e417170ebf54e06) 环境变量
+:::
+
+### UIC 新闻 {#bnu-hkbu-uic-news}
+
+<Route author="heimoshuiyu" example="/uic/news/top_stories" path="/uic/news/:sub" paramsDesc={['新闻子类']}>
+  |    头条     |    动态   |   深度   | 媒体聚焦 |
+  | ----------- | -------- | -------  |  ---- |
+  | top_stories | in_brief | features | media |
+</Route>
+
+### UIC 图书馆公告 {#bnu-hkbu-uic-lrc}
+
+<Route author="heimoshuiyu" example="/uic/lrc" />
+
+### UIC 理工科技学部 {#bnu-hkbu-uic-fst}
+
+<Route author="heimoshuiyu" example="/uic/fst" />
+
+### UIC 学生职业发展中心 {#bnu-hkbu-uic-career}
+
+<Route author="heimoshuiyu" example="/uic/career/ygtz" path="/uic/career/:tag" paramsDesc={['新闻子类']}>
+  |  ygtz  |  xsfx   |   gg    |    xngs    |   msjq  |  sjzxx |
+  | -----  |  -----  |  ----   |  --------- | -----  |  ------ |
+  | 中心新闻 | 学生风采 | 政府公告 | 就业质量报告 | 就业指南 | 升学讲座 |
+</Route>
+
```

Comment:
Do not edit the docs directly. All manual changes will be overwritten by each route.